recipe/build-llvm-openmp.sh

```diff
@@ -22,6 +22,9 @@ make install
 rm -f $PREFIX/lib/libgomp$SHLIB_EXT
 rm -f $PREFIX/lib/libiomp5$SHLIB_EXT
 
+# make a symlink
+ln -s $PREFIX/lib/libomp$SHLIB_EXT $PREFIX/lib/libiomp5$SHLIB_EXT
```

Comment:
I think this should go in https://github.com/conda-forge/_openmp_mutex-feedstock

Reply:
We need it on osx and linux, so should we create the mutex for osx too?

Reply:
Ah, that's okay then. I would just remove line 23.